bundle/manifests/3scale-operator.clusterserviceversion.yaml

```diff
@@ -1038,3 +1065,16 @@ spec:
   provider:
     name: Red Hat
   version: 0.0.1
+  webhookdefinitions:
+    - admissionReviewVersions:
+        - v1beta1
+        - v1beta2
+      containerPort: 443
+      conversionCRDs:
+        - products.capabilities.3scale.net
+      deploymentName: threescale-operator-controller-manager-v2
+      generateName: cproducts.capabilities.3scale.net
+      sideEffects: None
+      targetPort: 9443
+      type: ConversionWebhook
+      webhookPath: /convert
```

Comment:
This was added manually. For some reason, this is not generated and added automatically when generating the bundle. I still haven't found the reason for this, so if bundle are re-generated, this will be removed 🤔